perun-core/src/main/java/cz/metacentrum/perun/core/blImpl/GroupsManagerBlImpl.java

```diff
@@ -1567,7 +1567,15 @@ public void run() {
 				exceptionMessage = "Cannot synchronize group ";
 				log.error(exceptionMessage + group, e);
 				exceptionMessage+= "due to unexpected exception: " + e.getClass().getName() + " => " + e.getMessage();
-				throw e;
+				try {
+					throw e;
+				} catch (PrivilegeException pe) {
+					//FIXME Don't know why, but this catch block for PrivilegeException must be here
+					failedDueToException = true;
+					exceptionMessage = "Cannot synchronize group ";
+					log.error(exceptionMessage + group, pe);
+					exceptionMessage+= "due to unexpected exception: " + pe.getClass().getName() + " => " + pe.getMessage();
+				}
 			} finally {
```

Comment:
This is really wierd and we don't want to do this there. Remove Privilege Exception from the body of this method and also from throws, we don't want to use it there.